.goreleaser.yaml

```
@@ -0,0 +1,111 @@
+version: 2
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+
+builds:
+  - binary: scanner
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    ldflags:
+      - -X github.com/leaktk/scanner/cmd.Version={{.Version}} -X github.com/leaktk/scanner/cmd.Commit={{.Commit}}
```

```suggestion
      - -X github.com/leaktk/scanner/cmd.Version={{ .Version }} -X github.com/leaktk/scanner/cmd.Commit={{ .FullCommit }}
```

- format change
- using full commit